### PR TITLE
sngrep: new package

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: me / @diizzyy
Compile tested: ar71xx, mikrotik 750gl, r1122
Run tested: ar71xx, mikrotik 750gl, r1122

Description:
Add sngrep to repo

Signed-off-by: Daniel Engberg <daniel.engberg.lists@pyret.net>

### CODE REVIEW

**Comment by diizzyy** [comment]
@hnyman

File: net/sngrep/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sngrep is a tool for displaying SIP calls message flows from terminal.
+  It supports live capture to display realtime SIP packets and can also be used as PCAP viewer.
+endef
+
+CONFIGURE_ARGS += \
+	--with-pcre \
+	--enable-unicode \
+
+MAKE_FLAGS += \
+	CFLAGS="$(TARGET_CFLAGS)"
```
Comment (kdarbyshirebryant) [inline net/sngrep/Makefile]: I don't think it's required either.  I test built another package and the LEDE/Openwrt build system passes suitable make flags by default...it is after all cross compiling (or using a different build chain at the very least) everything by default :-)

Comment (diizzyy) [inline net/sngrep/Makefile]: This Makefile works fine also on Kirkwood so I'd propose to keep it as is and "fix it" later on if issues occur.

Comment (kdarbyshirebryant) [inline net/sngrep/Makefile]: I checked with a verbose build of your package, both with & without your 'MAKE_FLAGS' option.  In the case with 'MAKE_FLAGS', CFLAGS gets passed twice which is incorrect.  You should drop the MAKE_FLAGS from your commit.

**Comment by diizzyy** [comment]
Fixed in #3199

**Pull request closed without merging**